xml/Microsoft.VisualBasic.MyServices/RegistryProxy.xml

```diff
@@ -312,41 +312,7 @@
         <value>
           <see cref="T:Microsoft.Win32.RegistryKey" />
         </value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
```

Comment:
I removed the whole Remarks section, since this property is only supported on Win 95/98.